extensions/amp-live-list/0.1/live-list-manager.js

```diff
@@ -253,6 +267,18 @@ export class LiveListManager {
   }
 }
 
+/**
+ * Detects if a document has had transforms applied
+ * e.g. by a domain with signed exchange domain enabled.
+ * @param {!Document|!ShadowRoot} root
+ * @return {boolean}
+ */
+function isDocTransformed(root) {
+  const {documentElement} = (root.ownerDocument || root);
+  const transformed = documentElement.getAttribute('transformed');
+  return Boolean(transformed) && startsWith(transformed, TRANSFORMED_PREFIX);
```

Comment:
Aren't `!!` and `Boolean(x)` equivalent?

Reply:
Functionally the same, but they have a size difference and older browsers were very slow calling `Boolean` (because it's a function call and not an internal cast).

Reply:
I believe Closure Compiler rewrites calls to `Boolean` to `!!`
https://closure-compiler.appspot.com/home#code%3D%252F%252F%2520%253D%253DClosureCompiler%253D%253D%250A%252F%252F%2520%2540compilation_level%2520ADVANCED_OPTIMIZATIONS%250A%252F%252F%2520%2540output_file_name%2520default.js%250A%252F%252F%2520%253D%253D%252FClosureCompiler%253D%253D%250A%250A%252F**%250A%2520*%2520%2540param%2520%257Bstring%257D%2520name%250A%2520*%2520%2540return%2520%257Bboolean%257D%250A%2520*%252F%250Afunction%2520hello(name)%2520%257B%250A%2520%2520return%2520Boolean(name)%253B%250A%257D%250A%250A%252F**%250A%2520*%2520%2540param%2520%257Bstring%257D%2520name%250A%2520*%2520%2540return%2520%257Bboolean%257D%250A%2520*%252F%250Afunction%2520hello2(name)%2520%257B%250A%2520%2520return%2520!!name%253B%250A%257D%250A%250Awindow.hello%2520%253D%2520hello%253B%250Awindow.hello2%2520%253D%2520hello2%253B%250A%250A